src/demo/ui.jsx

```diff
@@ -432,6 +410,19 @@ class SpeechBubble extends React.Component {
 }
 
 class ActivityIntro extends React.Component {
+  onClickContinue = () => {
+    const state = getState();
+    if (state.loadTrashImages) {
+      setState({
+        currentMode: Modes.TrainingIntro,
```

Comment:
Should we have all transitions to a new `currentMode` go through our `toMode` helper?  Amongst other things, it seems to init the model.